org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/serializer/impl/PartialSerializer.java

```diff
@@ -336,6 +336,10 @@ public void setErrorAcceptor(ISerializationDiagnostic.Acceptor errorAcceptor) {
 		IFeatureInfo featureInfo = constraint.getFeatures()[owner.eClass().getFeatureID(feature)];
 		List<IConstraintElement> assignments = featureInfo.getAssignments();
 		if (assignments.size() != 1) {
+			ValueTransient valueTransient = transientValues.isValueTransient(owner, feature);
```

Comment:
What happens if there is more than one assignment? E.g. should the check for valueTransient be enhanced by `&& assignments.isEmpty()`?

Reply:
Good question. Have to think about that

Reply:
yes you are right. transient and zero assignments is fine. other cases really might need the sequencer to kick in

Reply:
updated the pr